Formula/mu.rb

```diff
@@ -49,13 +49,13 @@ class Mu < Formula
 
   def install
     mkdir "build" do
-      system "meson", *std_meson_args, ".."
+      args = std_meson_args + %W[
+        -Dlispdir=#{elisp}
+      ]
+      system "meson", *args, ".."
```

Comment:
```suggestion
      system "meson", *std_meson_args, "-Dlispdir=#{elisp}", ".."
```
Creating a new array just for one extra flag is a bit much.

Reply:
makes sense